src/cordova/plugin/android/java/org/outline/OutlinePlugin.java

```diff
@@ -259,13 +258,14 @@ public void onActivityResult(int request, int result, Intent data) {
     startVpnRequest = null;
   }
 
-  private int startVpnTunnel(final String tunnelId, final JSONObject config) throws Exception {
-    LOG.info(String.format(Locale.ROOT, "Starting VPN tunnel %s", tunnelId));
+  private int startVpnTunnel(final JSONObject jsonTunnelConfig) throws Exception {
```

Comment:
Let's revert to the more explicit interface we had before. 
I'd rather not have the TunnelConfig for now, as we don't configure anything else other than the proxy.
Also, I'm not sure the id belongs in a tunnel config if it's not used to configure the VPN.